src/nautilus_launch/launch/sim.launch

```diff
@@ -1,6 +1,6 @@
 <launch>
   <machine name="sim" address="sim" password="passwhat" env-loader="/root/ros_setup.sh"/>
-  <include file="$(find nautilus_launch)/launch/system.launch"/>

```

Comment:
nit: consider renaming the launch files to more descriptive names like `whole_system.launch` or `just_surface.launch`